.github/workflows/iroha2-dev-pr-title.yml

```
@@ -1,12 +1,12 @@
-name: Check PR Title
+name: I2::Dev::PR Title Check
```

```suggestion
name: I2::Dev::PR Title
```
The check comes from the first workflow element. We might be adding more, so please bear this in mind.

---

Yeah, my backup branch, which will give the next PR, includes it
https://github.com/s8sato/iroha/commit/37e57f9b8b9ec0d43cf1e8b8e0089a8fd2d8a3bf#diff-cf4b8b3df55d496290b6afbe73d45f29b99b5b543c2fe399125ec078a36b9c8e